Source/Pokens.Pokedex/Pokens.Pokedex.Office/Pages/Pokemons/ChangeStarter.cshtml.cs

```diff
@@ -12,21 +12,10 @@ public ChangeStarterModel(IPokemonService pokemonService)
         {
             this.pokemonService = pokemonService;
         }
-        public PokemonModel PokemonChosen { get; set; }
-
-        [BindProperty]
-        public CreatePokemonViewModel Pokemon { get; set; }
-
-        public void OnGet()
-        {
-            var id = HttpContext.Request.Query["id"];
-            this.PokemonChosen = this.pokemonService.GetAll().Where(p => p.Id == id).FirstOrDefault();
-        }
-        public IActionResult OnPost()
+        public IActionResult OnGet()
```

Comment:
verbul GET e un verb safe si idempotent... sau cel putin asa ar trebui sa fie (safe = nu modifica starea aplicatiei; idempotent = fiecare apel identic returneaza acelasi rezultat)

Faptul ca la GET noi schimbam IsStarter incalca principiile verbelor HTTP.
In cel mai rau caz, ar trebui sa facem asta la OnPost